web/docs/tql2/functions.md

```diff
@@ -126,7 +126,7 @@ Function | Description | Example
 Function | Description | Example
 :--------|:-------------|:-------
 [`as_secs`](functions/as_secs.md) | Converts a duration into seconds | `as_secs(42ms)`
-[`from_epoch_ms`](functions/from_epoch_ms.md) | Interprets a number as Unix time | `from_epoch_ms(1730234246123.456)`
+[`from_epoch`](functions/from_epoch.md) | Interprets a duration as Unix time | `from_epoch(10y)`
```

Comment:
A better example would be a reasonable timestamp (or maybe just an appropriately named field).

Reply:
I think something like `from_epoch(time_ms * 1ms)` would be a reasonable example.